_posts/2018-10-09-proposition-b.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Proposition B
+---
+
+Proposition B is about funding “community and cultural” facilities, including libraries, museums, art and creative
+spaces, which are “dedicated to the creation, exhibition, or preservation of art and creative expression.”
+
+Similar to Proposition A, the city wants to be able to:
+
+* Plan, design, construct, renovate and equip such community and cultural facilities
+* Buy land to support additional community and cultural facilities
+* Take out loans to support those activities
+
+In order to fund these activities, the city is proposing an issuance of <nobr><strong>$128 million</strong></nobr> in
+new bonds, noting that city property taxes will be used to fund the interest on those bonds.
+
+## What do “community and cultural facilities” include in Austin?
```

Comment:
Depending on how you feel about it in my prop PRs, I like a `## Questions` section so that longer questions like this can be h3s and not be quite so giant on the screen.

Reply:
Oh. But you changed the CSS since I looked. I feel less strongly about this, then. We should talk about it.